### ISSUE TITLE
[Feature] Accessibility audit and tasks

### ISSUE THREAD

**Issue by dimitur2204** [podkrepi-bg/frontend#1211]
### Description
Considering we are a platform for everyone, accessibility should be something we take seriously so anyone can access and **sufficiently interact** with our website!
This is an epic which should have all accessibility related tasks linked to.

Throughout the code there are these `A11Y TODO` comments indicating things that should be improved in terms of accessibility.
 
A good place to start would be fixing the Lighthouse report problems, but there is definitely more to think about than only the Lighthouse report.

This checklist here is an incredible resource about the job - https://www.a11yproject.com/checklist/

### Tasks
- [ ] Lighthouse accessibility problems

**Additional context**
Here is the current Lighthouse accessibility report on the homepage:
<img width="1720" alt="image" src="https://user-images.githubusercontent.com/61479393/204569013-0586c767-479b-4b75-bf0d-c53aba668d6b.png">

### Related issues
#339 - This adds a widget that helps impaired people to customize the website more easily, which is good, but does not solve accessibility problems by itself :)

**Comment by valkirilov** [comment]
Suggestion: Once we resolve all of the issues reported by the Lighthouse accessibility analysis we can move forward and integrate the following library, which can help us identify other a11y issues:
- https://github.com/dequelabs/axe-core-npm/tree/develop/packages/react
